src/photon/photon_scheduler.c

```diff
@@ -162,8 +162,14 @@ void reconstruct_object_task_lookup_callback(object_id reconstruct_object_id,
                                              task *task,
                                              void *user_context) {
   /* Recursively resubmit the task and its task lineage to the scheduler. */
-  CHECKM(task != NULL,
-         "No task information found for object during reconstruction");
+  /* TODO(swang): The following check will fail if an object was created by a
+   * put. Once we're able to reconstruct those objects as well, uncomment the
+   * check and remove the if-check and exit. */
+  /* CHECKM(task != NULL,
+         "No task information found for object during reconstruction"); */
+  if (task == NULL) {
```

Comment:
This will still occur in some cases (e.g., when the put happened on the driver).

I'd suggest making it fatal so that we know when it happens (the alternative is deadlock, right?).